app/util/substrateApi.js

```diff
@@ -1,16 +1,14 @@
-import { buildApi } from '@digicatapult/dscp-node'
+import { ApiPromise, WsProvider, Keyring } from '@polkadot/api'
 
 import env from '../env.js'
 import logger from '../logger.js'
 
 const { API_HOST, API_PORT } = env
 
-const { api, keyring: kr } = buildApi({
-  options: {
-    apiHost: API_HOST,
-    apiPort: API_PORT,
-  },
-})
+const provider = new WsProvider(`ws://${API_HOST}:${API_PORT}`)
+const api = new ApiPromise({ provider })
+
+api.isReadyOrError.catch(() => {})
```

Comment:
m? maybe a comment or someting e.g. // TODO implement err handler

Reply:
A think a comment would be good here. This is just to prevent unhandled promise rejection errors

Reply:
added `// prevent unhandled promise rejection errors`